### DISCUSSION TITLE
Where do I replace the files?

### DISCUSSION THREAD

**Discussion by lucr80** [pal1000/Realtek-UAD-generic#174] [General]
Solved, you can delete it.

**Comment by pal1000** [comment]
> What I want to know is which file contains the differences in the equalization of each driver?

RealltekAO folder handles equalization.

**Comment by lucr80** [comment]
I want to understand the Update Codec folder. In the device manager, mine always shows 9356. In this case, it updates to the latest version via the Update codec folder, but it shows the old name because it is signed, is that it?
